### DISCUSSION TITLE
how to get client?

### DISCUSSION THREAD

**Discussion by susanthenerd** [PrismarineJS/mineflayer#1653] [Q&A]
I need to get client from the bot.

**Comment by ghost** [comment]
> I need to get client from the bot.

You can write more clearly ?

**Comment by aesthetic0001** [comment]
what do you mean by "get client"?

**Comment by Zn10plays** [comment]
are you talking about the node-Minecraft-protocol client?
if that's what you are talking about, it would be bot._client 
where bot is your bot object.
